src/index.ts

```diff
@@ -53,6 +54,7 @@ const httpServer =
 const server = new ApolloServer({
   schema,
   formatError: (
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove and fix the root cause of the linting error

Reply:
what would be the type of error here it is showing error for every possilbe type I applied

Reply:
https://typescript-eslint.io/rules/no-explicit-any/

Try using the playground here where they give suggestions.